src/osx/menu_osx.cpp

```diff
@@ -324,6 +324,11 @@ bool wxMenu::HandleCommandUpdateStatus( wxMenuItem* item, wxWindow* senderWindow
     wxUpdateUIEvent event(menuid);
     event.SetEventObject( this );
 
+    if ( item )
+        event.SetIsCheckable(item->IsCheckable());
+    else
+        event.SetIsCheckable(true);
```

Comment:
Why is the event checkable when there is no item? Shouldn't this rather be
```suggestion
    event.SetIsCheckable(item && item->IsCheckable());
```
?